cmake/thirdparty/SetupThirdParty.cmake

```diff
@@ -115,3 +115,21 @@ if(CMAKE_GENERATOR STREQUAL "Unix Makefiles" OR CMAKE_GENERATOR STREQUAL "Ninja"
     blt_find_executable(NAME        ClangTidy
                         EXECUTABLES clang-tidy)
 endif()
+
+#------------------------------------
+# Code coverage
+#------------------------------------
+if (ENABLE_COVERAGE)
+    # find_executable requires that the executables be enabled
```

Comment:
For clarity, this comment should probably be something like:
> Attempt to find the executables associated with `gcov`, `lcov` and `genhtml`.
    This requires that the associate features are enabled.